backend/RoomPlanner/room-planner/src/main/java/edu/roomplanner/service/MapperService.java

```diff
@@ -0,0 +1,13 @@
+package edu.roomplanner.service;
+
+import edu.roomplanner.dto.RoomDto;
+import edu.roomplanner.entity.RoomEntity;
+import edu.roomplanner.entity.UserEntity;
+
+import java.util.List;
+
+public interface MapperService {
+
+    RoomDto mapToDto(RoomEntity roomEntity);
+    List<RoomDto> mapListToDto(List<UserEntity> roomEntityList);
```

Comment:
A more suitable name is 'mapEntityToDto', from the current name I understand that you map a list into a DTO